components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -398,6 +414,39 @@ public JsonObject getTopComments(String targetId, int likes)
 		}
 	}
 
+    public JsonObject getUserComment(String userId, String targetId) throws SocialActivityException {
+        JsonObject userComment = new JsonObject();
+        Connection connection = null;
+        PreparedStatement statement;
+        ResultSet resultSet;
+        String errorMsg = "Unable to retrieve comment for the user : " + userId;
+
+        try {
+            if (log.isDebugEnabled()) {
+                log.debug("Executing: " + USER_COMMENT_SELECT_SQL);
+            }
+            connection = DSConnection.getConnection();
+            statement = connection.prepareStatement(USER_COMMENT_SELECT_SQL);
+            statement.setString(1, targetId);
+            statement.setString(2, userId);
+            resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                JsonObject body = (JsonObject) parser.parse(resultSet.getString(Constants.BODY_COLUMN));
+                int activityId = resultSet.getInt(Constants.ID_COLUMN);
+                Activity activity = new SQLActivity(body);
+                activity.setId(activityId);
+                userComment = activity.getBody();
+            }
+            resultSet.close();
+        } catch (SQLException | DataSourceException e) {
+            String message = errorMsg + e.getMessage();
```

Comment:
Thanks @Rajith90  , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/0496a208d8bd8d3cd5f94dab8b0e8d67c540af73)